src/main/java/nl/tudelft/ti2206/group9/ShaftEscape.java

```diff
@@ -128,8 +128,8 @@ public static void exit() {
         stage.close();
         InternalTicker.stop();
         MainMenuScene.getAudioPlayer().stop();
-        GameScene.getAudioPlayer().resetSpeed();
-        GameScene.getAudioPlayer().stop();
+//      GameScene.getSoundtrackPlayer().resetSpeed();
```

Comment:
I temporarily disabled the increase of soundtrack methods, as I'm going to review it in #144.
(See comment above.)

<review